doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md

```diff
@@ -115,21 +115,7 @@ The 3rd party conponents listed above have to be cross-compiled before building
 
     Copy `${SDKTARGETSYSROOT}/usr/local/lib/libboost_*` object files to Edison's `/usr/local/lib` directory (e.g. using SSH, SCP, etc.).
 
-2. Install Avro for host and target. Also Avro depends on some Boost components. So they need to be installed too.
```

Comment:
I added a requirement to install Boost and Avro to the first chapter of the cross-compilation guide as a prerequisite:

> ... perform steps 1 and 2 of the [Linux guide] ... to install Boost and Avro components for your host machine.

I think it makes the guide easier to understand.